com.gigitux.youp.json

```diff
@@ -0,0 +1,40 @@
+{
+  "app-id": "com.gigitux.youp",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "youp",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=session-bus",
+    "--device=dri",
+    "--socket=pulseaudio",
+    "--filesystem=home"
```

Comment:
Do you really access to the whole home directory?

Reply:
Yes, it's necessary to allow upload files

Reply:
Ideally the app should use https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html for opening files. It would delegate passing over files to the portal and then this line can be dropped.

Reply:
You are right, but this app is a webview for whatsapp, I would like keep the same behaviour

Reply:
Ah, I haven't read the PR description closely enough. I'd change it to `xdg-download` if I were you.